searchJob/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+
+# Create your models here.
+
+
+class Company(models.Model):
+    company_name = models.CharField(max_length=100, primary_key=True)
+    profile_description = models.TextField()
+    establishment_date = models.DateField()
+    contact_number = models.CharField(max_length=12, blank=True, null=True)
+    company_type = models.TextChoices('company_type', 'FINTECH BIOTECH MEDTECH FOODTECH AUTOMATIVE SW HW')
```

Comment:
Remove it (FINTECH BIOTECH MEDTECH FOODTECH AUTOMATIVE SW HW)  to a consts that will be outside from the model